File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/viewrecent/fragment/binding/ViewRecentBinding.java
package ua.meugen.android.pdfcomposer.ui.activities.viewrecent.fragment.binding;

import java.util.List;

import ua.meugen.android.pdfcomposer.model.db.entity.PdfItemEntity;
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.binding.Binding;

/**
 * @author meugen
 */
public interface ViewRecentBinding extends Binding {

    void setupRecycler();

    void displayNoItems();

    void displayItems(List<PdfItemEntity> items);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/viewrecent/fragment/binding/ViewRecentBinding.kt
package ua.meugen.android.pdfcomposer.ui.activities.viewrecent.fragment.binding

import ua.meugen.android.pdfcomposer.model.db.entity.PdfItemEntity
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.binding.Binding

/**
 * @author meugen
 */
interface ViewRecentBinding : Binding {

    fun setupRecycler()

    fun displayNoItems()

    fun displayItems(items: List<PdfItemEntity>)
}
